ls1intum/Artemis
# `Plagiarism checks`: Fix plagiarism case verdict message

### Checklist
#### General
#### Server

### Motivation and Context
Plagiarism case verdict email didn't have its title placeholder resolved.

### Description
This PR adds missing `if` statement for resolving placeholder title of email sent when plagiarism case is resolved.

Surprisingly, there was a test case for that, which asserted that wrong value was returned by Artemis.

### Steps for Testing
Prerequisites:
- 1 Instructor
- 2 Students (on of them needs to be your TUM account, so that email is delivered)
- **test server with email support (example _test1_)**

1. Create an exercise (text is easiest to test). 
2. Submit two identical solution from two different student accounts.
3. From the instructor account go to the exercise management page -> check plagiarism.
4. Run checks and confirm displayed comparison.
5. Go to the new plagiarism case: Course -> Assessment -> Plagiarism cases.
6. Send a default notification to the student (yourself)
7. Select and apply plagiarism case verdict.
8. Verify that you got an email with proper title (not placeholder, see screenshots below)


### Testserver States
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test1)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test2)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test3)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test4)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test5)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test6)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test9)

### Review Progress
#### Code Review
#### Manual Tests

### Screenshots
Before:
![image](https://github.com/ls1intum/Artemis/assets/32958017/5163214a-db8d-4813-bc75-eaf2314e6777)

After:
![image](https://github.com/ls1intum/Artemis/assets/32958017/5573ef91-489b-455b-9e03-a7d35fb089cf)
